doc/tutorials/hires_howto.rst

```diff
@@ -0,0 +1,543 @@
+
+
+Spec1D
+------
+
+A summary of all the extracted sources is reported in an ASCII text with the
+same name as the spec1d fits file, but with the extension changed from ``.fits`` to ``.txt``.
+For this example, here are the first few lines of the file
+``Science/spec1d_HI.20151214.17593-SDSSJ0100+2802_HIRES_20151214T045314.323.txt``:
+
+.. code-block:: console
+
+   | order |          name | spat_pixpos | spat_fracpos | box_width | opt_fwhm |   s2n | wv_rms |
+   |    62 | OBJ0443-MSC01 |        66.2 |        0.443 |      3.00 |    0.804 | -0.01 |  0.157 |
+   |    61 | OBJ0443-MSC01 |       130.6 |        0.443 |      3.00 |    0.804 | -0.02 |  0.174 |
+   |    60 | OBJ0443-MSC01 |       197.2 |        0.443 |      3.00 |    0.804 | -0.02 |  0.179 |
+   |    59 | OBJ0443-MSC01 |       265.8 |        0.443 |      3.00 |    0.804 | -0.01 |  0.097 |
+   |    58 | OBJ0443-MSC01 |       336.5 |        0.443 |      3.00 |    0.804 |  0.01 |  0.105 |
+
+It shows a table with the PypeIt names of the extracted spectra in each order and all the associated
+information about the extraction. See :ref:`spec1d-extract_info` for a detailed description of this file.
+Each spectrum is given its own extension in the fits file, where the extension name is based
+on the object name identified in the file and the order number; e.g.,
+`OBJ0443-MSC01-ORDER0040`.
+
+You can plot the spectrum using :ref:`pypeit_show_1dspec`:
+
+.. code-block:: bash
+
+    pypeit_show_1dspec Science/spec1d_HI.20151214.17593-SDSSJ0100+2802_HIRES_20151214T045314.323.fits --exten 23
+
+The ``--exten 23`` argument specifies to use the 23rd extension in the fits file, which selects
+the spectrum in the order 40. This uses the `XSpecGUI`_ from the `linetools`_ package,
+and the result looks like this:
+
+.. figure:: ../figures/hires_spec1d_xgui.png
+
+   `XSpecGUI`_ produced by calling :ref:`pypeit_show_1dspec` for the order=40 spectrum.
+   The black line is the flux and the red line is the estimated error. In the window,
+   press ``?`` to open a webpage with the `XSpecGUI keystrokes`_ that help you navigate
+   through the spectrum.
+
+Another option for visualizing the 1D extracted spectrum is to use the `ginga`_ viewer. The call
+is simply:
+
+.. code-block:: bash
+
+    pypeit_show_1dspec Science/spec1d_HI.20151214.17593-SDSSJ0100+2802_HIRES_20151214T045314.323.fits --ginga
+
+which plots the spectrum in a tab of the `ginga`_ viewer and allows to select the different order spectra using a
+drop down menu, in addition to selecting other properties of the spectrum. Here is one exemple:
+
+.. figure:: ../figures/hires_spec1d_ginga.png
+
+
+Fluxing and co-adding
+=====================
+
+The flux calibration is performed by first creating a sensitivity function from the
+standard star observations. The sensitivity function is then applied to the science
+observations. See :ref:`fluxing` for more details.
+
+Generating a Sensitivity function
```

Comment:
We should include some big warnings about how hard it is to flux HIRES data (and probably a citation to the paper or two on the topic)

Reply:
Done it. Thanks.